code/__HELPERS/heap.dm

```diff
@@ -74,3 +73,43 @@
 /datum/heap/proc/List()
 	. = L.Copy()
 
+/// A specific implementation of a heap with the procs for dealing with pathfinding baked in, for maximum performance
+/datum/heap/path
+
+/datum/heap/path/Swim(index)
```

Comment:
Could you use `snake_case` for the procs?

Reply:
These aren't new but alright